code/src/index.css

```diff
@@ -11,3 +19,202 @@ code {
+  align-items: center;
+}
+
+.text-summary {
+  width: var(--container-width);
+}
+
+.form {
+  background: rgba(255, 255, 255, 0.7);
+  border: 0.5em solid rgb(233, 191, 7);
+  height: var(--container-height);
+  border-radius: 1.2rem;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  align-content: center;
+  flex-wrap: wrap;
+  justify-content: center;
+}
+
+.form-label, .form-button {
+  font-family: inherit;
+  margin-top: 1.2rem;
+}
+
+.form-label {
+  font-size: var(--font-size-big);
+  font-weight: 500;
+  margin-bottom: 0.4rem;
+}
+
+input[type="text"], input[type="email"] {
+  border: 0.16rem solid black;
+  border-radius: 1rem;
+  height: 2.5rem;
+  padding: 0 0.8rem;
+  font-size: var(--font-size-medium);
+  width: var(--container-width);
+  outline: none;
+}
+
+input[type="text"]:hover, input[type="email"]:hover {
+  background: rgb(235, 235, 235);
+}
+
+.button-container {
+  display: flex;
+  justify-content: space-around;
+  width: 100%;
+}
+
+.form-button {
+  display: inline-block;
+  font-size: var(--font-size-medium);
+  padding: 0.3rem 0.8rem;
+  margin-top: 1.5rem;
+  border: 0.16rem solid rgb(0, 0, 0);
+  border-radius: 2rem;
+  box-sizing: border-box;
+  text-decoration: none;
+  color: black;
+  background-color: rgb(233, 191, 7);
+  text-align:center;
+  animation-duration: 0.4s;
+  transform-origin: bottom;
+  cursor: pointer;
+}
+
+.form-button:hover {
+  background-color:#c69340;
+}
+
+.form-button:disabled {
+  background-color:#bda580;
+  border: 0.16rem solid rgb(138, 135, 135);
+  animation: none;
+  cursor: not-allowed;
+}
+
+.bouncy:hover {
+  animation-name: bouncy;
+  animation-timing-function: ease;
+}
+
+.error {
+  font-size: var(--font-size-small);
+  margin-left: 0.3rem;
+  margin-top: 0;
+  margin-right: 0;
+  margin-bottom: 0;
+}
+
+.p-text {
+  font-size: var(--font-size-medium);
+}
+
+@keyframes bouncy {
+  0%   { transform: translateY(0) }
+  50%  { transform: translateY(-5px) }
+  100% { transform: translateY(0) }
+}
+
+@media (max-width: 761px) {
+  :root {
+    --container-width: 80%;
+    --container-height: 32rem;
+    --inner-container-height: 8rem;
+    --font-size-small: 0.7rem;
+    --font-size-medium: 1rem;
+    --font-size-big: 1.3rem;
+    --font-header: 2.5rem;
+    --font-header-underscore: 1.5rem;
+    --margin-header: 0.8rem;
+  }
+}
+
+@media screen and (min-width: 762px), screen and (min-height: 650px) {
+  body {
+    background-image: url('./assets/background-beer.jpg');
+  }
+}
+
+@media screen and (min-width: 762px) {
+  :root {
+    --container-width-max: 600px;
+    --container-width: 80%;
+    --container-height: 35rem;
+    --font-size-small: 0.9rem;
+    --font-size-medium: 1.1rem;
+    --font-size-big: 1.4rem;
+    --font-header: 3rem;
+    --font-header-underscore: 1.8rem;
+    --margin-header: 1rem;
+  }
+}
+
+@media screen and (min-width: 1094px) {
+  body {
+    background-image: url('./assets/background-beer-large.jpg');
+  }
+
+  :root {
```

Comment:
So cool and inspiring to see how you worked with CSS-variables in this project! This is definitely something I will look into and see how I can implement them in my projects going forward!